data-raw/prepare_peer_group_net_zero_commitment.R

```diff
@@ -0,0 +1,73 @@
+#' Prepare data input for share of net zero committed companies in scorecard
+#'
+#' Prepare data input for share of net zero committed companies in scorecard
+#' based on PACTA for investors output files.
+#'
+#' @param total_portfolio Data frame. Contains processed input of the PACTA for
+#'   Investors calculation.
+#' @param peer_group Character. Peer group of the analysed portfolio.
+#' @param net_zero_targets Data frame. Contains information on which ISINs
+#'   belong to companies that have committed to SBTI net zero targets.
+#' @param peer_group_share_net_zero Data frame. Contains precalculated shares of
+#'   companies in the peer group portfolios that have SBTI net zero commitments.
+#'
+#' @return data.frame
+#' @export
+
+prepare_peers_net_zero_commitment <- function(total_portfolio,
```

Comment:
this should be migrated to `workflow.prep.PA2024CH`. As we discovered some of your changes seem to be reverted. Please check that all changes you made in previous commits wrt feedback are integrated.

Reply:
this function would be needed to be runned for any project and not only for PA2024CH so I understand we would want to keep it here?